src/components/Game/NoRunningGame.tsx

```diff
@@ -52,7 +52,7 @@ export const NoRunningGame = () => {
         <StyledButton onClick={() => onClickStartButton()}>
           Start New Game
         </StyledButton>
-        <StyledLabel>No Game running</StyledLabel>
+        <StyledLabel>No game running</StyledLabel>
```

Comment:
Steht so in seinem PDF

Reply:
Denke zwar nicht, dass das so drauf ankommt, aber für mich ists so auch ok ;)